assets/js/vue-apps/components/facet-choice.vue

```diff
@@ -0,0 +1,79 @@
+<script>
+import take from 'lodash/take';
+
+export default {
+    props: ['value', 'type', 'name', 'label', 'labelAny', 'options', 'optionLimit'],
+    data() {
+        return { isToggled: false };
+    },
+    computed: {
+        optionsToDisplay() {
+            if (this.shouldTruncate()) {
+                return this.isToggled ? this.options : take(this.options, this.optionLimit);
+            } else {
+                return this.options;
+            }
+        }
+    },
+    methods: {
+        shouldTruncate() {
+            return this.optionLimit && this.options.length >= this.optionLimit + 2;
+        },
+        fieldId(index) {
+            return `field-dynamic-${this.name}-${index}`;
+        }
+    }
+};
+</script>
+
+<template>
+    <fieldset class="ff-choice" v-if="options.length > 0">
+        <legend class="ff-label">
+            {{ label }}
+        </legend>
+         <ul class="ff-choice__list">
+             <li class="ff-choice__option" v-if="labelAny">
+                <input
+                    :type="type"
+                    :id="fieldId('any')"
+                    :name="name"
+                    value=""
+                    :checked="value === ''"
+                    v-on:input="$emit('input', $event.target.value)"
+                />
+                <label class="ff-choice__label" :for="fieldId('any')">
+                    {{ labelAny }}
+                </label>
+            </li>
+            <li class="ff-choice__option" v-for="(option, index) in optionsToDisplay" v-bind:key="option.value">
+                <input
+                    :type="type"
+                    :id="fieldId(index)"
+                    :name="name"
+                    :value="option.value"
+                    :checked="option.value === value"
+                    v-on:input="$emit('input', $event.target.value)"
+                />
+                <label class="ff-choice__label" :for="fieldId(index)">
+                    {{ option.label }}
+                </label>
+            </li>
+        </ul>
+
+        <button type="button"
+            class="btn-link"
+            v-if="shouldTruncate()"
+            @click='isToggled = !isToggled'
+        >
+            {{ isToggled ? 'See fewer options' : 'See more options' }}
```

Comment:
Maybe these phrases (and "Clear selection" below) could be passed in as parameters (eg. for translation)?